tests/python2-libraries/Dockerfile

```diff
@@ -0,0 +1,3 @@
+ARG intermediate_image
+FROM $intermediate_image
```

Comment:
Is this new?  I remember bashing my head in trying to make FROM statements configurable.

Reply:
Yep, this went live as of v17.06 I believe: 
https://docs.docker.com/engine/reference/builder/#understand-how-arg-and-from-interact
https://github.com/moby/moby/issues/18119